test/plugins/test_autobpm.py

```diff
@@ -1,7 +1,12 @@
+import importlib.util
+
 import pytest
 
 from beets.test.helper import ImportHelper, PluginMixin
 
+if not importlib.util.find_spec("librosa"):
```

Comment:
Nice. I initially thought 'why not simple `import librosa` in a try catch block' but why import it if it's not required in this module.

Note that we _always_ want to run these tests in CI: after this change, if librosa installation was accidentally removed from the CI, these tests would be skipped silently,

See [test_lyrics.py](https://github.com/beetbox/beets/blob/37a2ceccd1a0ca7f6e256d6bf23f92ea0ef6a38f/test/plugins/test_lyrics.py#L261) for an example condition to check whether the tests are run in CI or not.

Reply:
> Nice. I initially thought 'why not simple import librosa in a try catch block' but why import it if it's not required in this module.

I had that, and the CI told me not to do it that way :P

Reply:
Ah, the good ol' F401